src/api/supportRequest.ts

```diff
@@ -31,18 +31,24 @@ supportRequestRoutes.get('/getAll', async (req, res) => {
     }
     res.send(requests);
   } catch (error) {
+    logger.error('Unable to retrieve support requests: ', error);
     res.status(500).send('There Was An Internal Server Error');
   }
 });
 
 supportRequestRoutes.get('/getCount', async (req, res) => {
-  const ideaCount = await SupportRequest.count({ type: SupportRequestType.IdeaPitch, status: SupportRequestStatus.Pending });
-  const technicalCount = await SupportRequest.count({ type: SupportRequestType.TechnicalSupport, status: SupportRequestStatus.Pending });
+  try {
+    const ideaCount = await SupportRequest.count({ type: SupportRequestType.IdeaPitch, status: SupportRequestStatus.Pending });
+    const technicalCount = await SupportRequest.count({ type: SupportRequestType.TechnicalSupport, status: SupportRequestStatus.Pending });
 
-  res.json({
-    ideaCount,
-    technicalCount,
-  });
+    res.json({
+      ideaCount,
+      technicalCount,
+    });
```

Comment:
What could fail here?

Reply:
Forgot to include the lines above and below, but you added a `try/catch` around it, but I don't see anything that could fail 🤔

Reply:
Potentially these calls since they're hitting the DB: 
``` typescript
const ideaCount = await SupportRequest.count({ type: SupportRequestType.IdeaPitch, status: SupportRequestStatus.Pending });
     const technicalCount = await SupportRequest.count({ type: SupportRequestType.TechnicalSupport, status: SupportRequestStatus.Pending });
```

Reply:
Looking at their code it looks like it's set to return `0` in a failed call, but I guess a network connection could still fail. In that case though I would think it would have failed before this point. But better safe than sorry haha